frontend/coprs_frontend/coprs/views/webhooks_ns/webhooks_general.py

```diff
@@ -79,6 +80,22 @@ def decorated_function(copr, **kwargs):
 
     return decorated_function
 
+def add_webhook_history_record(webhook_uuid, user_agent='Not Set', builds_initiated_via_hook=None):
+    """
+    This methods adds info of an intercepted webhook to webhook_history db
+    along with the initiated build number(s).
+    """
+    if builds_initiated_via_hook is None:
+        log.debug("No build initiated. Webhook not loged to db.")
+        return
+
+    #For each build that was initiated by the webhook insert a record in db.
+    for build in builds_initiated_via_hook:
+        webhookRecord = models.WebhookHistory(webhook_uuid=webhook_uuid,
+                                              user_agent=user_agent, build_id=build)
+        db.session.add(webhookRecord)
+
+    db.session.commit()
```

Comment:
We are calling `db.session.commit()` after all usages of `add_webhook_history_record`, so we can remove the commit from here.

Reply:
fixed.